README.md

```diff
@@ -1,91 +1,90 @@
 # iidy (Is it done yet?) -- a CloudFormation CLI tool
 
-(Note: this is written assuming an Unbounce developer is the reader and we
-previously used Ansible for orchestrating CloudFormation.)
-
-`iidy` improves the developer experience with CloudFormation and
+iidy improves the developer experience with CloudFormation and
 CloudFormation templates.
 
-* It provides immediate, readable feedback about what CloudFormation
-  is doing and any errors it encounters.
-* Its single binary is simpler to install than Ansible's Python
-  dependencies.
-* It is far simpler to learn, understand, and use. There are fewer
-  moving parts and its commands map directly to CloudFormation's.
-* It requires less boilerplate files and directories. 2 or 3 files vs
-  5+ directories with 7+ files.
+* It provides immediate, readable feedback about what CloudFormation is doing
+  and any errors it encounters.
+* It is simple to learn, understand, and use.
+  parts, that map directly to CloudFormation's
```

Comment:
Some words are missing: "There are fewer moving parts and its commands map directly to CloudFormation's"